rundeck-authz/rundeck-authz-core/src/main/java/org/rundeck/core/auth/access/AuthActionsUtil.java

```diff
@@ -0,0 +1,144 @@
+package org.rundeck.core.auth.access;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.rundeck.core.auth.AuthConstants;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Utility methods for AuthActions interface
+ */
+public class AuthActionsUtil {
+
```

Comment:
Probably worth it to add a non public constructor to these Util classes so java doesnt add an implicit public constructor